frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx

```diff
@@ -0,0 +1,345 @@
+                title={text?.formSuccessTitle}
+                extra={[
+                  <p key="phar">{text.uploadThankyouText}</p>,
+                  <Divider key="divider" />,
+                  <Button
+                    type="primary"
+                    key="back-button"
+                    onClick={() => setShowSuccess(false)}
+                    shape="round"
+                  >
+                    {text.uploadAnotherFileLabel}
+                  </Button>,
+                  <Button
+                    key="page"
+                    onClick={() => navigate("/control-center")}
+                    shape="round"
+                  >
+                    {text.backToCenterLabel}
+                  </Button>,
+                ]}
+              />
+            </div>
+          )}
+          {!showSuccess && (
+            <>
+              <Row align="middle">
+                <Checkbox
+                  id="updateExisting"
+                  checked={updateExisting}
+                  onChange={() => {
+                    setUpdateExisting(!updateExisting);
+                  }}
+                >
+                  {text.updateExisting}
+                </Checkbox>
+              </Row>
+              <Card
+                style={{ padding: 0, minHeight: "40vh" }}
+                bodyStyle={{ padding: 0 }}
+              >
+                <Space align="center" size={32}>
+                  <img src="/assets/data-download.svg" />
+                  <p>{text.templateDownloadHint}</p>
+                  <Select
+                    placeholder="Select Attributes..."
+                    className="multiple-select-box"
+                    onChange={handleChange}
+                    mode="multiple"
+                    allowClear
+                  >
+                    {attributes.map((f, fI) => (
+                      <Option key={fI} value={f.id}>
+                        {f.name}
+                      </Option>
+                    ))}
+                  </Select>
+                  <Button
+                    loading={loading}
+                    type="primary"
+                    onClick={downloadTemplate}
+                    shape="round"
+                  >
+                    {text.download}
+                  </Button>
+                </Space>
+                <Space align="center" size={32}>
+                  <img src="/assets/data-upload.svg" />
+                  <p>{text.uploadDataLabel}</p>
+                  <Select
+                    placeholder="Select Form..."
+                    value={formId}
+                    onChange={handleChange}
+                    className="custom-select"
+                  >
+                    {forms.map((f, fI) => (
+                      <Option key={fI} value={f.id}>
+                        {f.name}
+                      </Option>
+                    ))}
+                  </Select>
+                  <AdministrationDropdown />
```

Comment:
We also don't need this

Reply:
<AdministrationDropdown /> only this or the entire <Space>....</Space>?

Reply:
Please keep in mind that this upload, doesn't have any relations with forms. Please re-check all of them